_layouts/project_extended.html

```diff
@@ -0,0 +1,92 @@
+        <div class="col-xs-12 col-md-6">
+           <a href="https://indyautonomouschallenge.com/timeline"> <img class="about__group-image" src="/assets/images/projects/2020/indy-timeline.jpg"></a>
+        </div>
+    </div>
+    
+    <br><br>
+
+    <a href="https://static1.squarespace.com/static/5da73021d0636f4ec706fa0a/t/5e604e57e4fc8938f7a8ed33/1583369815480/University+of+Pittsburgh+White+Paper.pdf">Read Our Whitepaper!</a>
+
+    <br><br>
+
+    {% include centerheading.html title='Project Leads: ' %}
+    <div class="row">
+        <div class="about__person col-xs-12 col-sm-6 col-lg-4">
+            <img src="/assets/images/about/current/nayana-suvarna.jpg">
+            <div class="about__person-title">Nayana Suvarna</div>
+            <div class="about__person-position">Project Co-Lead</div>
+        </div>
+        <div class="about__person col-xs-12 col-sm-6 col-lg-4">
+            <img src="/assets/images/about/current/prem-bharatia.jpg">
+            <div class="about__person-title">Prem Bharatia</div>
+            <div class="about__person-position">Project Co-Lead</div>
+        </div>
+        <div class="about__person col-xs-12 col-sm-6 col-lg-4">
+            <img src="/assets/images/about/alumni/officer-andrew.jpg">
+            <div class="about__person-title">Andrew Saba</div>
+            <div class="about__person-position">Project Mentor</div>
+        </div>
+        <div class="about__person col-xs-12 col-sm-6 col-lg-4">
+            <img src="/assets/images/projects/2020/mentors/josh-spisak.jpg">
+            <div class="about__person-title">Joshua Spisak</div>
+            <div class="about__person-position">Project Mentor</div>
+        </div>
+        <div class="about__person col-xs-12 col-sm-6 col-lg-4">
+            <img src="/assets/images/about/advisors/advisor-dickerson.jpg">
+            <div class="about__person-title">Samuel Dickerson</div>
+            <div class="about__person-position">Project Advisor</div>
+        </div>
+    </div>
+
+    <br>
+
+    {% include centerheading.html title='Project Members: ' %}
+    <h5>Elissa Wilton, Matthew Sivaprakasam, Michael Finnegan, Michael Sohnen, Mike Hayden Morgan Visnesky, Shane Raible</h5>
+
+    <br><br>
+
+    {% include centerheading.html title='How to Join' %}
+    <p>
+        Whether you are just starting out or you have years of experience, you can find a place to contribute to our team. The best way to get involved is to join our slack. No experience is nessesasry but please note that this project is a rather large time commitment.
```

Comment:
Whether you are just starting out or you have years of experience, you can find a place to contribute to our team. The best way to get involved is to join our slack. No experience is necessary but please note that this project is a rather large time commitment. (fix typo)

Reply:
Fixed